### PR TITLE
Mongoid 6 support

### PR DESCRIPTION
**Pull request by gurix**
Hey Hey, I toke @narutosanjiv changes, updated the readme and  implemented a proper test matrix for travis. Please don't hesitate to comment and I hope you accept this PR.

### CODE REVIEW

**Comment by rodrigopinto** [comment]
Nice! Could you squash the commits. The @narutosanjiv could become only one commit and yours another one.

File: .travis.yml [hunk 1]
```diff
@@ -5,9 +5,20 @@ rvm:
   - 2.1.8
   - 2.2.4
   - 2.3.0
+  - 2.3.1
```
Comment (rodrigopinto) [inline .travis.yml]: here could be only `2.3.1` because it's a minor fix.

File: README.md [hunk 2]
```diff
@@ -8,6 +8,13 @@ mongoid-rspec provides a collection of RSpec-compatible matchers that help to te
 
 ## Installation
 
+### With Mongoid 6.x
+
+Use mongoid-rspec [3.1.0][mongoid6]
+
+    gem 'mongoid-rspec', '~> 3.1.0'
+
+
```
Comment (rodrigopinto) [inline README.md]: Do we need this? Or we could use the version 3.1.0 with mongo 5 and 6?

**Comment by gurix** [comment]
@rodrigopinto Changed everything according your suggestion. I hope the squashing is as you expected, I never did that before.

**Comment by ivanovaleksey** [comment]
Hi guys!
Can it be merged?

**Comment by 90yukke** [comment]
I'm closing this PR as beta support of mongoid 6 is already in master. See https://github.com/mongoid-rspec/mongoid-rspec/pull/179

Also I don't think that HEAD of mongoid-rspec should support 2 versions of mongoid at time.

**Pull request closed without merging**